include/vapor/FlowParams.h

```diff
@@ -25,13 +25,16 @@ enum class FlowDir : int
     FORWARD     = 0,
     BACKWARD    = 1,
     BI_DIR      = 2 
-}; 
+};
 
 
 
 class PARAMS_API FlowParams : public RenderParams 
 {
 public:
+    
+    enum RenderType { RenderTypeStream, RenderTypeSamples};
+    enum GlpyhType { GlpyhTypeSphere, GlpyhTypeArrow };
```

Comment:
I did originally but it resulted in too many explicit casts.

Reply:
Can you elaborate a bit more? Why is an explicit cast required, and why a traditional enum result in less explicit casts?

Reply:
Explicit casts are required because, like you said, enum classes result in stricter type safety. Traditional enums are essentially just constants.

Reply:
Can you elaborate why you will need too many explicit casts, no matter new or old style enums? 

Regarding safety, it's exactly because old-style enums permit an implicit cast, thus there's more chance the program goes wrong without the awareness of the programmer, isn't it? So the new-style enum is safer because its usage is more limited, isn't it?